File: QTD2.Infrastructure/Model/Task Requalification/TaskQualificationCreateOptions.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.Task_Requalification
{
    public class TaskQualificationCreateOptions
    {
        public int TaskId { get; set; }

        public int EmpId { get; set; }

        public int? EvaluationId { get; set; }

        public int[] EvaluatorIds { get; set; }

        public DateTime? TaskQualificationDate { get; set; }

        public string TaskQualificationEvaluator { get; set; }

        public DateTime? DueDate { get; set; }

        public bool CriteriaMet { get; set; }

        public string Comments { get; set; }
    }
}

File: QTD.SPA/src/app/_DtoModels/TaskQualification/TaskQualificationCreateOptions.ts
export class TaskQualificationCreateOptions {
  taskId!: any;
  empId!: any;
  evaluationId!: any;
  taskQualificationDate?: Date;
  taskQualificationEvaluator: string;
  dueDate: Date;
  criteriaMet: boolean;
  comments: string;
  evaluatorIds:any[] = [];
}
